packages/sparseqr/test_sparseqr.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+from pyodide_test_runner import run_in_pyodide
+
+
+@pytest.mark.driver_timeout(40)
+@run_in_pyodide(packages=["scipy,sparseqr"])
+def test_scipy_linalg(selenium):
+    import numpy
+    import scipy.sparse.linalg
+    import sparseqr
+    from numpy.testing import assert_allclose
+
+    # QR decompose a sparse matrix M such that  Q R = M E
+    #
+    M = scipy.sparse.rand(10, 10, density=0.1)
+    Q, R, E, rank = sparseqr.qr(M)
+    print(
+        abs(Q * R - M * sparseqr.permutation_vector_to_matrix(E)).sum()
+    )  # should be approximately zero
```

Comment:
Could you change these `print` statements to test assertions?
Also ideally this could be split into several separate tests.

Reply:
MMhh .. After a second read this morning I realized I had completely botched the test file. The new commit should be closer to something that would work, although I wasn't really able to test it. Using pytest, I get the following error :

FAILED sparseqr/test_sparseqr.py::scipy_linalg[firefox] - ModuleNotFoundError: No module named 'sparseqr'